app/src/main/java/org/oppia/app/drawer/NavigationDrawerFragmentPresenter.kt

```diff
@@ -97,16 +98,21 @@ class NavigationDrawerFragmentPresenter @Inject constructor(
     if (previousMenuItemId != menuItemId && menuItemId != 0) {
       when (NavigationDrawerItem.valueFromNavId(menuItemId)) {
         NavigationDrawerItem.HOME -> {
+          drawerLayout.closeDrawers()
           val intent = HomeActivity.createHomeActivity(activity, internalProfileId)
           fragment.activity!!.startActivity(intent)
-          fragment.activity!!.finish()
+        }
+        NavigationDrawerItem.OPTIONS -> {
+          drawerLayout.closeDrawers()
+          fragment.activity!!.startActivity(OptionsActivity.createOptionsActivity(fragment.requireContext(), internalProfileId))
```

Comment:
Open Options from nav-drawer, click back -> home appears and now open drawer again, you will see `Options` highlighted. Instead, home should be highlighted. Once you fix this, also write test case to make sure this works correctly.

Reply:
+1

Reply:
I will remove those changes. When we press back from Option Page it is directly going to Profile page instead of HomePage to fix it I added this code. Let's file an issue and fix in other PR.